arch/riscv/README.md

```diff
@@ -1,10 +1,9 @@
 # Building RISC-V Target with Cmake #
 
 > **Warning**
-> We cannot detect rvv support at runtime, running the rvv code on a no-rvv target is a risk. Users should disable the rvv when the target does not support it. 
+> `hwcap` detect for rvv is only supported on kernel 6.5+.
 >
-> We will have a better solution when the kernels update `hwcap` or `hwprobe` for risc-v.
-
+> If you are using a lower version, it would fallback to compile-time detection, running the rvv code on a no-rvv target is a risk. Users should disable the rvv when the target does not support it.
```

Comment:
Now I see what you are getting at, but it is a little confusing.

How about something like this? (personally I'd also prefer typing RVV in caps, like we do with other instruction sets)
```
Runtime rvv detection (using `hwcap`) requires linux kernel 6.5 or newer.

When running on older kernels, we fall back to compile-time detection, potentially this can cause crashes if rvv is enabled at compile but not supported by the cpu.
Therefore if older kernel support is needed, rvv should be disabled if the target cpu does not support it.
```

Reply:
Thanks! Modified.